### PR TITLE
frontend logo, menu updates

### PR DESCRIPTION
**Pull request by achen2401**
- update App logo
- remove the text, “leaf”, under logo in header
- hide `PHI` option from header `roles` menu item
- minor styling fix to display when no data to show in visualization

### CODE REVIEW

**Review by rpkyle — APPROVED** [review]
Thanks @achen2401, looks good to me. I will approve the PR, but suggest waiting for @ivan-c to comment also before merging.

File: src/ui-client/src/containers/Header/Header.tsx [hunk 1]
```diff
@@ -56,8 +56,8 @@ class Header extends React.PureComponent<Props> {
             <Navbar id={`${c}-container`} className="d-flex justify-content-between mb-3">
                 <div className={`${c}-content-side`}>
                     <div className={`${c}-title`} >
-                        <img alt="leaf-logo" className="logo" src={process.env.PUBLIC_URL + '/images/logos/apps/leaf.png'} />
-                        <div className="title">leaf</div>
+                        <img alt="leaf-logo" className="logo" src={process.env.PUBLIC_URL + '/images/logos/apps/logo_light.png'} />
+                        {/* <div className="title">leaf</div> */}
```
Comment (ivan-c) [inline src/ui-client/src/containers/Header/Header.tsx:60]: Is this commented-out section still needed?
```suggestion

```

File: src/ui-client/src/containers/Attestation/Attestation.tsx [hunk 2]
```diff
@@ -149,9 +149,9 @@ class Attestation extends React.PureComponent<Props, State> {
 
                     {/* Top logo row (Leaf, ITHS, CD2H) */}
                     <div className={`${c}-leaf-logo-wrapper`}>
-                        <img alt='leaf-logo' className={`${c}-leaf-logo`} src={process.env.PUBLIC_URL + '/images/logos/apps/leaf.png'} />
+                        <img alt="leaf-logo" className="logo" src={process.env.PUBLIC_URL + '/images/logos/apps/logo_dark.png'} />
                         <div className={`${c}-title`}>
-                            leaf
+                            {/* leaf */}
```
Comment (ivan-c) [inline src/ui-client/src/containers/Attestation/Attestation.tsx]: Is this commented-out section still needed?
```suggestion
```

**Review by ivan-c — APPROVED** [review]
Looks good, thank you!

File: src/ui-client/src/containers/Header/Header.tsx (hunk 1 above)
Comment (achen2401) [reply src/ui-client/src/containers/Header/Header.tsx:60]: thank you Ivan. I removed it as suggested.

**Pull request merged**